src/app/components/CopyToClipboard/index.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC, useCallback, useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+import Tooltip from '@mui/material/Tooltip'
+import ContentCopyIcon from '@mui/icons-material/ContentCopy'
+import IconButton from '@mui/material/IconButton'
+
+const clipboardTooltipDuration = 2000
+
+type CopyToClipboardProps = {
+  value: string
+}
+
+export const CopyToClipboard: FC<CopyToClipboardProps> = ({ value }) => {
+  const { t } = useTranslation()
+  let timeout: number | undefined
```

Comment:
This would be nice to wrap inside of `useRef` otherwise if rerender happens it would be set to undefined again.